ngraph/test/runtime/ie/unit_test.manifest

```diff
@@ -1540,3 +1540,27 @@ onnx_controlflow_loop_infinite
 # unsupported dynamic ops
 onnx_dyn_shapes_reduce_max_dynamic_input_rank_negative_axis
 IE_GPU.range_v4_trunc_inputs
+
+# not implemented jet on CPU and GPU plugins
+IE_CPU.evaluate_1D_gather_elements_3_indices_int32
```

Comment:
If you remove the `IE_*.` prefix and leave the test name only in this file, such line will disable this test for both plugins.

Reply:
On GPU GatherElements will be implemented much after CPU so it's better to keep this separation. But thanks for the information, I didn't know that.